forge-gui/res/cardsfolder/upcoming/jan_jansen_chaos_crafter.txt

```diff
@@ -0,0 +1,11 @@
+Name:Jan Jansen, Chaos Crafter
+ManaCost:R W B
+Types:Legendary Creature Gnome Artificer
+PT:3/3
+K:Haste
+SVar:TrigToken:DB$ Token | TokenScript$ c_1_1_a_construct

```

Comment:
This TrigToken line seems unneeded?

Reply:
Think that was something from a script I used as a template that I forgot to clean up. Looks like I should be able to rip it out without any consequences. Going to double check now.

Reply:
Confirmed unnecessary and removed.